mne/stats/cluster_level.py

```diff
@@ -1116,23 +1108,30 @@ def permutation_cluster_1samp_test(X, threshold=None, n_permutations=1024,
 
     Returns
     -------
-    T_obs : array of shape [n_tests]
+    T_obs : array, shape (n_tests,)
         T-statistic observed for all variables
     clusters : list
         List type defined by out_type above.
     cluster_pv : array
         P-value for each cluster
-    H0 : array of shape [n_permutations]
+    H0 : array, shape (n_permutations,)
         Max cluster level stats observed under permutation.
 
     Notes
     -----
-    Reference:
-    Cluster permutation algorithm as described in
-    Maris/Oostenveld (2007),
-    "Nonparametric statistical testing of EEG- and MEG-data"
-    Journal of Neuroscience Methods, Vol. 164, No. 1., pp. 177-190.
-    doi:10.1016/j.jneumeth.2007.03.024
+    If ``n_permutations >= 2 ** (n_samples - (tail == 0)) - 1``,
```

Comment:
If it's a 2-tailed test then a `[-1, 1, -1]` flip is identical in max-cluster-size to a `[1, -1, 1]` flip, as you'll get the same largest cluster, just with opposite sign. If it's a 1-tailed test then they are not equivalent, as you will git different clusters (because you only look at the positive or negative half)